android/src/main/java/com/guichaguri/trackplayer/module/MusicModule.java

```diff
@@ -113,6 +113,7 @@ private void waitForConnection(Runnable r) {
 
         // Capabilities
         constants.put("CAPABILITY_PLAY", PlaybackStateCompat.ACTION_PLAY);
+        constants.put("CAPABILITY_TOGGLE_PLAY_PAUSE", PlaybackStateCompat.ACTION_PLAY);
```

Comment:
There is also an [ACTION_PLAY_PAUSE](https://developer.android.com/reference/android/support/v4/media/session/PlaybackStateCompat.html#ACTION_PLAY_PAUSE) in Android

Reply:
OK. I'll update.